Source/WebKit/WebProcess/Extensions/Bindings/JSWebExtensionWrapper.h

```diff
@@ -196,9 +196,13 @@ NSString *serializeJSObject(JSContextRef, JSValueRef, JSValueRef* exception);
 
 #ifdef __OBJC__
 
-@interface JSValue (ThenableExtras)
+@interface JSValue (WebKitExtras)
```

Comment:
Why is this all private with underscore prefixes?

Reply:
These are internal helper methods that we don't plan to expose as API.